android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -58,6 +58,7 @@ fun purchaseProduct(
     oldSku: String?,
     prorationMode: GoogleProrationMode?,
     type: String,
+    isPersonalizedPrice: Boolean?,
```

Comment:
i've toyed in my head with whether isPlayPricePersonalized or something that indicates this is google-specific would be better naming... and it feels even more important maybe in the hybrids. thoughts?

Reply:
might end up wanting to do oldGoogleProductId and googldProrationMode too... 

also, speaking of... looks like oldsku can be changed to oldProductId. ignore if you're doing that in the future..